pkg/messaging/rabbitmq/pubsub_test.go

```diff
@@ -4,21 +4,25 @@
 package rabbitmq_test
 
 import (
+	"context"
 	"fmt"
 	"testing"
 
+	"github.com/gogo/protobuf/proto"
 	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/pkg/messaging/nats"
```

Comment:
Importing Nats for RabbitMQ tests?

Reply:
This was because they had same errors, I'll copy the errors to rabbitmq